apps/client/src/theme/_v2Styles.scss

```diff
@@ -60,6 +60,6 @@ $text-body-size: 15px;
 
 @keyframes blink {
   50% {
-    opacity: 0;
+    opacity: 40%;
```

Comment:
I think having the opacity only go down to 40% here makes it to subtle here.
I think this feature should intenionally be super annoying since it's purpose is to rise awarness from the speake to the timer

Reply:
That is a good point.
The reason I took it up is because the button disappears in the interface, which I find a bit strange.

Do you think there should be different settings for editor and timer? Or would 20% everywhere work?

Reply:
In my oppinion 20% everywhere would work fine